### DISCUSSION TITLE
PROBLEMS!!!!!!!!!1

### DISCUSSION THREAD

**Discussion by prabhatbhatiaa** [community/community#60255] [GitHub Education]
### Hi everyone,

I am trying to avail the developer pack but it shows me error. I have my school I-card which shows I am in XI. I also uploaded my pic with school id. Still I get errors. Please help me anyone

**Comment by GhostOf0days** [comment]
Try uploading a screenshot of your cost of attendance bill if it has your school’s logo on it.

**Comment by lavkush-maurya** [comment]
If you're encountering errors while trying to avail a developer pack, I recommend reaching out to the support team or customer service of the platform or service you are trying to access. They should be able to provide you with the necessary assistance to resolve the issue.

Make sure to provide them with detailed information about the error message you're receiving and any steps you have already taken. They will be better equipped to troubleshoot the problem and guide you through the necessary steps to resolve it.

If you're not sure how to contact the support team, you can visit the platform's website and look for a "Contact Us" or "Support" page. There, you should find information on how to reach out to them, whether it's through email, a support ticket system, or a live chat.

I hope you're able to get the help you need and successfully avail the developer pack!

**Comment by samus-aran** [comment]
Hi @prabhat-codez 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Ensure your evidence is accurate to the evidence type selected 👍 

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
